cmd/authtoken/main.go

```diff
@@ -23,6 +23,10 @@ var (
 	configPath string
 )
 
+const (
+	aksAADApplicationID = "6dae42f8-4368-4678-94ff-3960e28e3630"
+)
+
```

Comment:
why move the azure specific const to the main?

Reply:
I have moved back the const to `pkg/authtoken/providers/azure/azure_msi.go`, and keep the default scope value logic there too.